### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by youmari**
## In this branch I implemented the following : 

- Config a webpack config file 
- Create an array of objects containing items data
- Add a function to create  all elements of the todo list and display them 
- Style the to-do list  and add icons 
- Add a good descriptive Readme file

### CODE REVIEW

**Comment by youmari** [comment]
## note ⚠️ 
### the lighthouse error because it was required to generate index.html inside source so lighthouse doesn't find index.html in the root 
- Runtime error encountered: The page did not paint any content. Please ensure you keep the browser window in the foreground during the load and try again. (NO_FCP)

**Review by Dipeshtwis — COMMENTED** [review]
# Status: Invalid ❌

While You have done a great job in your project, I am not able to review it as you are not using the correct linter file. Please fix the issue mentioned below and then request for code review

## Reason

- [x] You are using the incorrect `linters.yml` file here. For javascript, you should use [this linter](https://github.com/microverseinc/linters-config/tree/master/javascript) file, and no need of `.hintrc` file as well. Please remove that.

### Please make sure to submit a code review when you are done with the changes that were requested

**Don't worry you still have same number of requests left as previous.**


Cheers and Happy Coding :clap: :clap: :clap: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

**_Dipesh Kumar_**

File: README.md [hunk 1]
```diff
@@ -1,59 +1,43 @@
...
-- Technologies used
+- HTML
+- CSS
+- JAVASCRIPT
+- WEBPACK
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://youmari.github.io/todo-list)
```
Comment (Dipeshtwis) [inline README.md:17]: [optional]

- The live link is not working. Please fix it or remove the section from the README.

**Review by Dipeshtwis — APPROVED** [review]
# Hi @youmari, 

### Status: Approved ✔️

:+1:  Professional repo, with good commit message and PR title, keep up the good work :muscle:
:+1: The project setup is well organized. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comment under the review_**


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**